website/SQL/create.sql

```diff
@@ -0,0 +1,76 @@
+CREATE DATABASE IF NOT EXISTS ZoomFront;
```

Comment:
The actual name of the database is generally left up to the programmer configuring it. 

Let's say I was a sysadmin running your service, and I wanted to do a zero-downtime upgrade on a live version of your application. Some update you pushed out removes the `Pass` column on `Users`. Without the ability to spin up a second database, my process as a sysadmin _has_ to be putting down the original application, pulling the git repo, running your schema migration, verifying the data integrity, THEN putting the thing up. Not being able to configure the DB name restricts my workflow to being synchronous, when what I really want to be doing is spinning up a second instance of your app pointing to a staging database.